sql/hive/src/test/scala/org/apache/spark/sql/sources/HadoopFsRelationTest.scala

```diff
@@ -126,61 +126,60 @@ abstract class HadoopFsRelationTest extends QueryTest with SQLTestUtils with Tes
     } else {
       Seq(false)
     }
-    // TODO: Support new parser too, see SPARK-26374.
-    withSQLConf(SQLConf.LEGACY_TIME_PARSER_ENABLED.key -> "true") {
```

Comment:
Can we leave this `withSQLConf(SQLConf.LEGACY_TIME_PARSER_ENABLED.key -> "false") {`? That's going to clearly show the diff and technically this configuration isn't removed yet.

Reply:
done